src/main/java/org/minions/devfund/katerinaanzoleaga/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,280 @@
+    String getBestMovie() {
+        double maxRating = 0;
+        int index = -1;
+        Movie theMovie;
+        for (Movie movie : this.movieList) {
+            if (movie.getRating() > maxRating){
+                maxRating = movie.getRating();
+                index = this.movieList.indexOf(movie);
+            }
+        }
+        theMovie = this.movieList.get(index);
+        return theMovie.getName();
+
+    }
+
+
+    /**
+     * Fill Movies list from file.
+     * @param fileName
+     */
+    void fillMoviesFromFile(String fileName) {
+        try {
+            File moviesFile = new File(fileName);
+            Scanner scnr = new Scanner(moviesFile);
+            String[] array1;
+            while (scnr.hasNext()) {
+                String aLine = scnr.nextLine();
+                array1 = aLine.split(",");
+
+                String[] actors = new String[1];
+                actors[0] = array1[0].trim();
+                for (int j = 1; j < array1.length; j++) {
+                    int i = this.movieExists(array1[j].trim());
+                    if (i != -1) {
+                        int a = this.actorExists(array1[0].trim());
+                        if (a == -1) {
+                            Actor anActor;
+                            anActor = new Actor();
+                            anActor.setName(array1[0].trim());
+                            anActor.addMovie(this.movieList.get(i));
+                            this.movieList.get(i).addActor(anActor);
+                            this.actorList.add(anActor);
+                        }
+                        else {
+                            this.actorList.get(a).addMovie(this.movieList.get(i));
+                        }
+                    }
+                    else
+                        this.addMovie(array1[j].trim(), actors);
+                }
+            }
+            scnr.close();
+        }
+        catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+    /**
+     * Fills the movies rationgs from the file.
+     * @param fileName
+     */
+    void fillRatingsFromFile(String fileName) {
+        try {
+            File ratingsFile = new File(fileName);
+            Scanner scnr = new Scanner(ratingsFile);
+            String[] array1;
+            while (scnr.hasNext()) {
+                String aLine = scnr.nextLine();
+                array1 = aLine.split("	");
+                String aRating = array1[1];
+                if (! aRating.equals("critic_score")){
+                    double rating;
+                    try {
+                        rating = Integer.parseInt(aRating);
+                        this.addRating(array1[0], rating*1.0);
+                    }
+                    catch (NumberFormatException e) {
+                        e.printStackTrace();
+                    }
+                }
+            }
+        }
+        catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)